decidim-generators/Gemfile.lock

```diff
@@ -435,22 +435,22 @@ GEM
     fog-local (0.8.0)
       fog-core (>= 1.27, < 3.0)
     formatador (1.1.0)
-    foundation_rails_helper (4.0.0)
-      actionpack (>= 4.1, < 7.0)
-      activemodel (>= 4.1, < 7.0)
-      activesupport (>= 4.1, < 7.0)
-      railties (>= 4.1, < 7.0)
-    geocoder (1.7.5)
+    foundation_rails_helper (4.0.1)
+      actionpack (>= 4.1, < 7.1)
+      activemodel (>= 4.1, < 7.1)
+      activesupport (>= 4.1, < 7.1)
+      railties (>= 4.1, < 7.1)
+    geocoder (1.8.0)
```

Comment:
The `geocoder` update from 1.7 to 1.8 has broken [some specs](https://github.com/decidim/decidim/runs/6858938810?check_suite_focus=true):


```bash
rspec ./spec/lib/map/provider/geocoding/here_spec.rb:35 # Decidim::Map::Provider::Geocoding::Here with geocoder request stubs #coordinates requests the nominatim API with correct parameters
rspec ./spec/lib/map/provider/geocoding/here_spec.rb:46 # Decidim::Map::Provider::Geocoding::Here with geocoder request stubs #address requests the nominatim API with correct parameters
```

This is probably related to this [change in Geocoder](https://github.com/alexreisner/geocoder/blob/master/CHANGELOG.md#180-2022-may-17): 

> Update :here lookup to use API version 7. Query options are different, API key must be a string (not an array). See API docs at https://developer.here.com/documentation/geocoding-search-api/api-reference-swagger.html (thanks github.com/Pritilender).

I think we have a couple of options: 

1. To leave it at 1.7, make this release and look it up with calm
2. To check locally if It's still working and to fix the spec   

Whatever you prefer is OK for me @ahukkanen. I can check it tomorrow.

Reply:
I have locked it to 1.7.x at 6dead7eb65364158a1e742985f816aeb0ab22e4e.